docs/resources/integration_grant.md

```diff
@@ -12,7 +12,7 @@ description: |-
 ## Example Usage
 
 ```terraform
-resource snowflake_integratin_grant grant {
+resource snowflake_integration_grant grant {
```

Comment:
This file is autogenerated, you need to change this in examples/resources/integration_grant/resource.tf and then run `make docs` to update it.

Reply:
ok thanks